spec/expectation-spec.coffee

```diff
@@ -249,105 +260,105 @@ describe 'Brainstem Expectations', ->
       loader.setup(name: "projects", include: ["anything"])
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", {})
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
     it "should treat strings and numbers the same when appropriate", ->
-      expectation = new Brainstem.Expectation("projects", { only: "1" }, manager)
+      expectation = new Expectation("projects", { only: "1" }, storageManager)
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", only: 1)
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", only: "1")
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
     it "should treat null, empty array, and empty object the same", ->
-      expectation = new Brainstem.Expectation("projects", { filters: {} }, manager)
+      expectation = new Expectation("projects", { filters: {} }, storageManager)
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", filters: null)
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", filters: {})
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", {})
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", filters: { foo: "bar" })
       expect(expectation.loaderOptionsMatch(loader)).toBe false
 
-      expectation = new Brainstem.Expectation("projects", {}, manager)
+      expectation = new Expectation("projects", {}, storageManager)
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", filters: null)
       expect(expectation.loaderOptionsMatch(loader)).toBe true
 
-      loader = new Brainstem.CollectionLoader(storageManager: manager)
+      loader = new CollectionLoader(storageManager: storageManager)
       loader.setup(name: "projects", filters: {})
```

Comment:
These are perfect examples of specs that could be nicely cleaned up with `beforeAll` blocks that Jasmine 2 provides us.